pkg/aws/aws_client/image.go

```diff
@@ -26,15 +27,17 @@ func (client *AWSClient) DescribeImage(imageIDs []string, filters ...map[string]
 	filterInput := []types.Filter{}
 	for _, filter := range filters {
 		for k, v := range filter {
+			copyKey := k
 			awsFilter := types.Filter{
```

Comment:
Is it required to add it: `copyKey := k`?

Reply:
Yes. It is required, otherwise the key pointer will be override.